src/Stratis.Bitcoin.Features.PoA/Voting/VotingManager.cs

```diff
@@ -296,6 +300,33 @@ public bool AlreadyVotingFor(VoteKey voteKey, byte[] federationMemberBytes)
             return false;
         }
 
+        public Poll CreatePendingPoll(DBreeze.Transactions.Transaction transaction, VotingData votingData, ChainedHeader chainedHeader, List<Vote> pubKeysVotedInFavor = null)
+        {
+            Poll poll = null;
+
+            // Ensures that highestPollId can't be changed before the poll is committed.
+            this.PollsRepository.Synchronous(() =>
+            {
+                poll = new Poll()
+                {
+                    Id = this.PollsRepository.GetHighestPollId() + 1,
+                    PollVotedInFavorBlockData = null,
+                    PollExecutedBlockData = null,
+                    PollStartBlockData = new HashHeightPair(chainedHeader),
+                    VotingData = votingData,
+                    PubKeysHexVotedInFavor = pubKeysVotedInFavor ?? new List<Vote>() {  }
+                };
+
+                this.polls.Add(poll);
+
+                this.PollsRepository.AddPolls(transaction, poll);
+
+                this.logger.LogDebug("New poll was created: '{0}'.", poll);
```

Comment:
Can we change this a log Info so that if anythign goes awry we can easier ask masternode user's for their logs?

Reply:
Done